tyk-pro/configs/tyk_analytics.conf

```diff
@@ -27,7 +27,7 @@
     "portal_root_path": "/"
   },
   "home_dir": "/opt/tyk-dashboard",
-  "use_sharded_analytics": true,
+  "use_sharded_analytics": false,
```

Comment:
Why? Will it affect existing deployments?

Reply:
This currently is breaking the analytics in the log browser. With the current pumps in the pump.conf. This only applies for multi-tenants and requires the selective pump to work. Issue is not a lot of users use multi tenancy and that is required in their licenses so we thought it is best to turn it off.